apps/verify/behaviours/validate-case-details.js

```diff
@@ -15,8 +16,10 @@ module.exports = superclass => class extends superclass {
 
     const imaResult = casesJson.find(
       obj => obj.cepr === cepr && obj['date-of-birth'] === dob);
-    req.form.values['duty-to-remove-alert'] = imaResult['duty-to-remove-alert'];
-    req.sessionModel.set('duty-to-remove-alert', imaResult['duty-to-remove-alert']);
+    if(_.isEmpty(imaResult) === false) {
+      req.form.values['duty-to-remove-alert'] = imaResult['duty-to-remove-alert'];
+      req.sessionModel.set('duty-to-remove-alert', imaResult['duty-to-remove-alert']);
```

Comment:
Do you need to do both of these?

Reply:
Tested without the first one that sets the form values and it works. Originally had it in there to test that the `duty-to-remove-alert` was being set properly for it to be picked up in the form later.

Reply:
Thanks for updating